dev-scripts/directory-structure-maker/directory-structure-maker.sh

```diff
@@ -0,0 +1,95 @@
+mkdir -p ./ENFORCER/\$HOME
+mkdir -p ./ENFORCER/bin
+
+# Creates folder structure in ROUTER
+mkdir -p ./ROUTER/\$HOME/
+mkdir -p ./ROUTER/etc/envoy
+mkdir -p ./ROUTER/etc/ssl/certs
+
+ADAPTER_IMAGE_ID=$(docker ps | grep 'wso2/choreo-connect-adapter' | awk '{ print $1 }')
+ENFORCER_IMAGE_ID=$(docker ps | grep 'wso2/choreo-connect-enforcer' | awk '{ print $1 }')
+ROUTER_IMAGE_ID=$(docker ps | grep 'wso2/choreo-connect-router' | awk '{ print $1 }')
+
+echo "Prepareing Choreo Connect folder structure..."
+
+# Copies files from Adapter container
+docker cp $ADAPTER_IMAGE_ID:home/wso2/adapter           ./ADAPTER/\$HOME
+mv ./ADAPTER/\$HOME/adapter                             ./ADAPTER/\$HOME/adapter-$ARC_VALUE
+docker cp $ADAPTER_IMAGE_ID:home/wso2/check_health.sh   ./ADAPTER/\$HOME
+docker cp $ADAPTER_IMAGE_ID:home/wso2/LICENSE.txt       ./ADAPTER/\$HOME
+docker cp $ADAPTER_IMAGE_ID:home/wso2/conf              ./ADAPTER/\$HOME
+docker cp $ADAPTER_IMAGE_ID:home/wso2/security          ./ADAPTER/\$HOME
+docker cp $ADAPTER_IMAGE_ID:bin/grpc_health_probe       ./ADAPTER/bin
+mv ./ADAPTER/bin/grpc_health_probe                      ./ADAPTER/bin/grpc_health_probe-$ARC_VALUE
+
+# Copies files from Enforcer container
+docker cp $ENFORCER_IMAGE_ID:home/wso2/conf            ./ENFORCER/\$HOME
+docker cp $ENFORCER_IMAGE_ID:home/wso2/lib             ./ENFORCER/\$HOME
+docker cp $ENFORCER_IMAGE_ID:home/wso2/security        ./ENFORCER/\$HOME
+docker cp $ENFORCER_IMAGE_ID:home/wso2/check_health.sh ./ENFORCER/\$HOME
+docker cp $ENFORCER_IMAGE_ID:home/wso2/LICENSE.txt     ./ENFORCER/\$HOME
+docker cp $ENFORCER_IMAGE_ID:bin/grpc_health_probe     ./ENFORCER/bin
+mv ./ENFORCER/bin/grpc_health_probe                    ./ENFORCER/bin/grpc_health_probe-$ARC_VALUE
+
+# Copies files from Router container
+docker cp $ROUTER_IMAGE_ID:home/wso2/interceptor        ./ROUTER/\$HOME/
+docker cp $ROUTER_IMAGE_ID:home/wso2/wasm               ./ROUTER/\$HOME/
+docker cp $ROUTER_IMAGE_ID:LICENSE.txt                  ./ROUTER/
+docker cp $ROUTER_IMAGE_ID:/etc/envoy/envoy.yaml        ./ROUTER/etc/envoy/
+docker cp $ROUTER_IMAGE_ID:/etc/ssl/certs/ca-certificates.crt  ./ROUTER/etc/ssl/certs
+
+# Copies dockerfiles to the relevant folder
+cp $CC_REPO_PATH/adapter/src/main/resources/Dockerfile                          ./ADAPTER
+cp $CC_REPO_PATH/adapter/src/main/resources/Dockerfile.ubuntu                   ./ADAPTER
+cp $CC_REPO_PATH/enforcer-parent/enforcer/src/main/resources/Dockerfile         ./ENFORCER
+cp $CC_REPO_PATH/enforcer-parent/enforcer/src/main/resources/Dockerfile.ubuntu  ./ENFORCER
+cp $CC_REPO_PATH/router/src/main/resources/Dockerfile                           ./ROUTER
+cp $CC_REPO_PATH/router/src/main/resources/Dockerfile.ubuntu                    ./ROUTER
+
+# Creates update directory txt
+touch ./updates/product.txt
+echo "choreo-connect-"$CC_VERSION >> ./updates/product.txt
+
+echo "Choreo Connect folder structure prepared successfully!"
```

Comment:
I think we use remove-container script in other places. Check for usages.

Reply:
Fixed with: https://github.com/wso2/product-microgateway/pull/2875/commits/a876184e5ff849920343eef2801e399fa2aa211c